SQL/migrate/V053__preference_asfx_instrument.sql

```diff
@@ -0,0 +1,7 @@
+--
+-- Set default 'On' for 'asfx_instruments'
+-- 
+
+UPDATE `ss13_player_preferences` SET `asfx_togs` = `asfx_togs` | 128;
+ALTER TABLE `ss13_player_preferences`
+	ALTER `asfx_togs` SET DEFAULT '128';
```

Comment:
I am not sure if that default value is correct.

Reply:
```suggestion
	ALTER `asfx_togs` SET DEFAULT '255';
```

Reply:
Corrected this following discussion with Skull here <https://discordapp.com/channels/131477462461841409/131489773268238336/736665658875445328> by removing the ALTER line, as the default is set in-code already in /datum/preferences